src/main/resources/assets/admin/common/js/ui/time/Picker.ts

```diff
@@ -8,24 +8,41 @@ import {AppHelper} from '../../util/AppHelper';
 import {FormEl} from '../../dom/FormEl';
 import {StyleHelper} from '../../StyleHelper';
 import {SelectedDateChangedEvent} from './SelectedDateChangedEvent';
+import {DateTime} from '../../util/DateTime';
+
+/**
+ * Not required for pickers
+ * Should only be used when th default value is needed in children
+ */
+export class DefaultValuePickerBuilder {
+    defaultValue: any;
+
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
This seems like a place that we want to have `any`  since we could use the picker for any value in the future